combinetf2/fitter.py

```diff
@@ -954,6 +956,11 @@ def _compute_yields_noBBB(self, compute_norm=False, full=True):
             elif self.indata.systematic_type == "normal":
                 normcentral = norm * ernorm + logsnorm
```

Comment:
can normcentral also get negative?

Reply:
presumably since nexpcentral is the sum of normcentral along its last dim, yes, but I didn't see this causing problems (maybe because it's not used in `_compute_nll_components`?)